src/components/ItemList.vue

```diff
@@ -16,11 +10,13 @@
         </transition-group>
       </div>
     </transition>
+    <newsListNav :page="page" :maxPage="maxPage" :type="type" v-if="displayedPage > 0"></newsListNav>
```

Comment:
Can we use `kebab-case` here to keep it consistent with the rest of the project?